### ISSUE TITLE
Adding graph measures to pre-existing data output. 

### ISSUE THREAD

**Issue by joshbear** [neurosutton/connectivity#8]
Be useful to be able to run gaf.save_long_format_results but specify only specific graph measures to be included. Then, if the specified output file already exists, it would be updated (instead of overwritten) to add a column for the new graph measure(s) calculated.

**Comment by neurosutton** [comment]
Agree. That was the original intention of the pandas version of the function, though the function got unwieldy. I'll think through some strategies before tackling this.

**Comment by joshbear** [comment]
Thinking about this a bit more in practical use cases. It might be helpful to have the file be written as the processes complete (so, for example, if you are running a range of thresholds and had to stop halfway through for any reason, the first half would still be saved). Similarly, if you then ran it again using the same parameters, it would check for an existing file and only start at the subject/threshold needed.

**Comment by neurosutton** [comment]
I think I addressed this second point by overwriting at each loop of save_long_format_results.

Still working on the lookup function.

**Comment by neurosutton** [comment]
Thinking of a nested dictionary solution to the lookup issue.
